gradle/libs.versions.toml

```
@@ -3,18 +3,18 @@ commonmark = "0.22.0"
 composeDesktop = "1.7.0-rc01"
 detekt = "1.23.6"
 dokka = "1.9.20"
-idea = "2024.2.3"
-intelliJPlatformBuild = "242.23339.11"
+idea = "243.20331"
```

This should align with the platform build below

```suggestion
idea = "243.19420.21"
```

---

This is intentionally misaligned.

---

Ok, is this supposed to be aligned afterwards then? I'm not clear why it's misaligned

---

Yes, it is not aligned at the moment, because the icons generation, theme gen etc. is requiring us to depend on maven artifacts that I would have to build and publish separately (they don't use the installers). But we don't care about them for now. When EAP with LafFlowService comes out (which will be soon, we are going to switch to this eap and align those.